src/components/venues/Jazzbar/EntranceExperience/EntranceExperience.scss

```diff
@@ -1,6 +1,44 @@
 @import "scss/constants.scss";
 
 .venue-entrance-experience-container {
+  .toast {
+    position: absolute;
+    max-width: none;
+    width: 40%;
+    margin: auto;
+    background: #3ce348;
+    color: $black;
+
+    .toast-header {
+      background: #208a28;
```

Comment:
you don't gather your colours in some colors file?

Reply:
yes sure forgot this one